pkg/network/ebpf/c/protocols/events.h

```diff
@@ -54,9 +55,9 @@
                 }                                                                                       \
                                                                                                         \
                 if (use_ring_buffer) {                                                                  \
-                    perf_ret = bpf_ringbuf_output(&name##_batch_events, batch, sizeof(batch_data_t), 0);\
+                    perf_ret = bpf_ringbuf_output_with_telemetry(&name##_batch_events, batch, sizeof(batch_data_t), 0);\
```

Comment:
Let's make sure USM is OK with this.

Reply:
added for debugging purposes. Will remove